engine/src/main/kotlin/world/gregs/voidps/engine/client/EncodeExtensions.kt

```diff
@@ -17,11 +17,12 @@ import java.util.*
  */
 
 /**
- * A chat box message to display
- * @param type The message type
- * @param tile The tile the message was sent from
- * @param name Optional display name?
- * @param text The chat message text
+ * Sends a message to the client with specific chat settings and formatting.
+ *
+ * @param text The content of the message that will be sent.
+ * @param type The type of chat message to display, default is ChatType.Game.
+ * @param tile The tile ID associated with the message, default is 0.
+ * @param name Optional name associated with the message, default is null.
```

Comment:
These param descriptions are better than the original, but arguably neither should exist.